jupyter-org-client.el

```diff
@@ -648,6 +654,16 @@ inserted without modification as the result of a code block."
   "Return a comment `org-element' with VALUE."
   (list 'comment (list :value value)))
 
+(defun jupyter-org-export-or-pandoc (type value params)
```

Comment:
Can we rename this to `jupyter-org-export-block-or-pandoc`? It makes it slightly more obvious what the function is doing.

Reply:
Fixed.